src/table/dresult.jl

```diff
@@ -0,0 +1,17 @@
+struct DResult
```

Comment:
Oops, that's a prototype file I made for one of the ideas from the todo list, it's not referenced anywhere though, so I'll remove it for now.
Basically it's supposed to represent a result of an operation that is first applied per chunk and then aggregated using some function. For example: map is first applied per chunk and then result gets aggregated using vcat when you call fetch on it.
It would be useful if you never wanted to collect the result into memory and just save it to the disk, but apart from map I didn't really find any useful scenarios for usage of this

Reply:
Removed for now, topic is in todo